libraries/stdlib/src/kotlin/random/Random.kt

```diff
@@ -267,10 +267,12 @@ public abstract class Random {
      *
      * @sample samples.random.Randoms.defaultRandom
      */
-    companion object Default : Random() {
-
+    companion object Default : Random(), Serializable {
+        private const val serialVersionUID = 6449517406788167720L
```

Comment:
I still don't get what bad can happen if we set `serialVersionUID = 1L` in all such classes?